### ISSUE TITLE
Auto-start servers on join

### ISSUE THREAD

**Issue by JT-Potato** [Minehut/Meta#752]
### Feature request

**Checklist**
- [x] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [x] This is not a bug report, support request or plugin request
- [x] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [x] This issue does not contain any sensitive information 

**Platform**

In-game: Java

**Describe the feature**

Instead of joining the lobby when the server is offline, the server is automatically started when a player wants to join it, bypassing the lobby. (Starting the server could take a while, so would a limbo mode work (where the player is sent to a lobby that is supposed to take minimal resources, just so that they don't time out from the server))

**How the feature is useful**

This would make it easier for server members to join servers. Yes, even though it's 1 command, it can create the illusion that Minehut servers are 24/7, without having the hardware permanently dedicated to the server.

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by xenitrogen** [comment]
I don't think this is needed perhaps just a message when you join the lobby saying "The server you were trying to connect to is offline! To start it use /join (server name)"

**Comment by JT-Potato** [comment]
> I don't think this is needed perhaps just a message when you join the lobby saying "The server you were trying to connect to is offline! To start it use /join (server name)"

I guess, but it's more manual and breaks the illusion of a 24/7 server. I've heard that Minehut gets a lot of traffic from people having to go into the lobby, but I think they might get more traffic from google searches for "free 24/7 minecraft server"

**Comment by TrueReassembly** [comment]
Minehut does supply 24/7 servers for MHU, starting a server when they connect to it would be a lot more load on the servers, I connect through offline servers a lot on my server list and i dont really care to get dumped in the lobby
